components/grid/templates/filter.md

```diff
@@ -305,6 +308,100 @@ For an example with the CheckboxList Filter, see the [Custom Data]({%slug grid-c
 
 ![Custom Filter Menu Template with Checkboxes](images/custom-filter-menu-checkboxes.png)
 
+## Filter Menu Buttons Template
+
+By default, the Filter Menu renders the `Filter` and `Clear` buttons. You can customize or remove them entirely by using the `FilterMenuButtonsTemplate` tag. 
+
+The template receives a `context` of type `FilterMenuTemplateContext` that provides the following members:
+
+* `FilterDescriptor`—the object that describes the column filter. By default, the column filter has two filters: one for the type and another for the name of the field. You can modify the column filter by:
+   * Adding more filters to the `FilterDescriptors` collection.
+   * Changing the `LogicalOperator` (`AND` by default).
+   * Using the `FilterDescriptor` to create a custom button that applies a predefined filter. 
+* `FilterAsync`—applies the filters defined in the Filter Menu to the Grid component.
+* `ClearFilterAsync`—clears the applied filters.
+
+>caption Implementing a button with a predefined filter.
+
+````CSHTML
+@* Customize the buttons in the Filter Menu
+
+@using Telerik.DataSource
+
+<TelerikGrid Data="@GridData"
+             Pageable="true"
+             Sortable="true"
+             FilterMode="@GridFilterMode.FilterMenu">
+    <GridColumns>
+        <GridColumn Field="Name" Title="Product Name" />
+        <GridColumn Field="Price">
+            <FilterMenuButtonsTemplate Context="filterContext">
+                <TelerikButton OnClick="@(async _ => await filterContext.FilterAsync())">Filter </TelerikButton>
+                <TelerikButton OnClick="@(() => SetPredefinedFilterAsync(filterContext))">Select >= 55</TelerikButton>
+                <TelerikButton OnClick="@(() => ClearFilterAsync(filterContext))">Clear</TelerikButton>
+            </FilterMenuButtonsTemplate>
+        </GridColumn>
+        <GridColumn Field="@nameof(Product.Released)" />
+        <GridColumn Field="@nameof(Product.Discontinued)" />
+    </GridColumns>
+</TelerikGrid>
+
+@code {
+    private async Task SetPredefinedFilterAsync(FilterMenuTemplateContext filterContext)
+    {
+        var compositeFilterDescriptor = filterContext.FilterDescriptor;
+        compositeFilterDescriptor.FilterDescriptors.Clear();
+        compositeFilterDescriptor.LogicalOperator = FilterCompositionLogicalOperator.Or;
+
+        compositeFilterDescriptor.FilterDescriptors.Add(new FilterDescriptor()
+            {
+                Member = nameof(Product.Price),
+                MemberType = typeof(int),
+                Operator = FilterOperator.IsGreaterThanOrEqualTo,
+                Value = 55
+            });
```

Comment:
Extra tab?

Reply:
It should not be, its aligned by VS, but I will fix that